database/migrations/20220110072020_jobs.js

```diff
@@ -0,0 +1,33 @@
+exports.up = async function (knex) {
+  await knex.raw(`create extension if not exists "uuid-ossp"`);
+  return await knex.schema.createTable("jobs", function (table) {
+    table
+      .uuid("uuid")
+      .notNullable()
+      .primary()
+      .defaultTo(knex.raw("uuid_generate_v4()"));
+    table.string("user_uuid", 255).notNullable();
+    table.string("company_name", 255).notNullable();
+    table.string("title", 255).notNullable();
+    table.string("experience", 255).notNullable();
+    table.jsonb("location").nullable();
+    table.text("description").nullable();
+    table.specificType("requirements_attachments", "TEXT[]").nullable();
```

Comment:
Not clear why requirements_attachments is TEXT[], can't it be boolean or jsonb?

Reply:
Also @ShubhamJoshi-1994 @rajeshn95 do we need these many columns, one jsonb would be enough for entire job profile, and take only those which are needed for "filtering". Even search can be managed via jsonb.

Reply:
> 

This column has an array of attached resume/cv file UUIDs @rjvim

Reply:
@mahesh-muttinti If it's just fileuuids, we can't store in jsonb, like an array? why TEXT[] complication? I am guessing its an array of strings, which is json

Reply:
yes, it is an array of strings, then should I make it to `JSON` data type?
`table.json("requirements_attachments").nullable();` Is this correct?

Reply:
@maheshmuttinti https://www.compose.com/articles/faster-operations-with-the-jsonb-data-type-in-postgresql/